app/models/manageiq/providers/embedded_terraform/automation_manager/ibmcloud_credential.rb

```diff
@@ -0,0 +1,58 @@
+class ManageIQ::Providers::EmbeddedTerraform::AutomationManager::IbmcloudCredential < ManageIQ::Providers::EmbeddedTerraform::AutomationManager::CloudCredential
```

Comment:
The naming of the file and casing of the class doesn't look right.

I suggest we follow what we use elsewhere:

https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/container_manager.rb

`class ManageIQ::Providers::IbmCloud::ContainerManager < ManageIQ::Providers::Kubernetes::ContainerManager`

in file: `providers/ibm_cloud/container_manager.rb`

So, `IbmCloudCredential` in `automation_manager/ibm_cloud_credential.rb`